crates/sncast/src/starknet_commands/account/import.rs

```diff
@@ -80,6 +80,12 @@ pub async fn import(
     };
     let private_key = &SigningKey::from_secret_scalar(*private_key);
 
+    // let account = if let Some(name) = account {
+    //     name
+    // } else {
+    //     generate_default_account_name()
+    // };
+
```

Comment:
We shouldn't keep comments unless they are related with todo or there is a good reason to keep it in the codebase.